pkg/models/capturetraffic.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import "time"
+
+type TrafficCaptureParams struct {
+	Path             string
+	ProxyPort        uint32
+	AppCmd           string
+	AppContainer     string
+	AppNetwork       string
+	Delay            uint64
+	BuildDelay       time.Duration
+	Ports            []uint
+	Filters          *TestFilter
+	EnableTele       bool
+	PassThroughHosts []Filters
```

Comment:
any reason to make this all global?

Reply:
In the future, if we would need to use the CaptureTraffic, it would be easier to access the global struct to easily pass the vars from command to service layer